be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/validators/CoinValidator.scala

```diff
@@ -65,8 +63,8 @@ case object CoinValidator extends MessageDataContentValidator {
     */
```

Comment:
Same as above:  Line 62 comments should be corrected to match the new convention